doc/news/changes/minor/20170726AlbertoSartori

```diff
@@ -0,0 +1,3 @@
+Fixed: FunctionParser uses shared_ptr instead of unique_ptr in order to fix compilation issues with gcc-5.4.
```

Comment:
Yes, that was the original entry that I forgot to update. The issue may be moot.